tests/regression/arista/arista_test.py

```diff
@@ -313,6 +320,14 @@ def testStandardTermV4(self):
         self.assertIn(expected, str(acl), str(acl))
 
         self.naming.GetNetAddr.assert_has_calls([mock.call('SOME_HOST')])
+    
+    @capture.stdout
+    def testSIPUsesInt(self):
+        self.naming.GetServiceByProto.return_value = ['5060']
+        pol = policy.ParsePolicy(GOOD_HEADER + GOOD_TERM_8, self.naming)
+        acl = arista.Arista(pol, EXP_INFO)
+        self.assertIn('5060', str(acl))
+        self.assertNotIn('sip', str(acl))
```

Comment:
```suggestion
        self.assertNotIn('sip', str(acl))
        print(acl)
```

Would be good to get output capture on this case.